docs/modernization/InkToolbar.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
@normesta / @mcleanbyron Could you review this new documentation for Ink Toolbar?